client/python/armada_client/client.py

```diff
@@ -59,6 +61,16 @@ def get_job_events_stream(
         )
         return self.event_stub.GetJobSetEvents(jsr)
 
+    def unmarshal_event_response(self, event: event_pb2.EventStreamMessage) -> Event:
```

Comment:
This should be a static function?  It doesn't seem to depend on self for anything.